sdks/java/testing/tpcds/build.gradle

```diff
@@ -18,6 +18,7 @@
 
 plugins { id 'org.apache.beam.module' }
 applyJavaNature(
+        publish: false,
```

Comment:
@iemejia @aromanenko-dev Do we want to publish this package during release?

Reply:
Hmm, good catch. We do this for Nexmark but I don't think it's needed as published artifacts. 
@iemejia whyt?

Reply:
I think publishing them is better to enable other parties to reproduce it outside of Beam. Also adding an extra module probably does not add too much extra weight to the release process.

Reply:
Ack. So the only difference is I added 2 TODO to indicate addressing warngings.